test/not_null.t.cpp

```diff
@@ -392,10 +799,10 @@ struct ETLessWeirdPtr // // element_type and type T differ
 using std::is_same;
 #else
 template<class T, class U>
-struct is_same { enum { value = false }; };
+struct is_same { enum dummy { value = false }; };
```

Comment:
Why introduce a name for the enum?

Reply:
It fixed -Wunnamed-type-template-args warnings introduced in PR #192